app/views/directives/centreGender.html

```diff
@@ -29,11 +29,16 @@
       <td ng-bind="heartbeatreceived"></td>
       <td ng-bind="ooc"></td>
     </tr>
-    <tr ng-if="prebooking !== undefined">
+    <tr>
```

Comment:
prebookings aren't live yet, the frontend should be backwards compatible for a few versions, we probably need to consider implementing semver sometime soon, so you'll need the ng-if to remain there